webportal/src/app/cluster-view/services/services.component.js

```diff
@@ -35,24 +36,13 @@ const serviceViewHtml = serviceViewComponent({
 
 const loadServices = () => {
   loading.showLoading();
-  $.ajax({
-    //url: `${webportalConfig.restServerUri}/api/v1/job`,
-    //type: 'GET',
-    success: (data) => {
-      loading.hideLoading();
-      if (data.error) {
-        alert(data.message);
-      } else {
-        $('#service-table').html(serviceTableComponent({
-          services: data      
-        }));
-      }
-    },
-    error: (xhr, textStatus, error) => {
-      const res = JSON.parse(xhr.responseText);
-      alert(res.message);
-    }
+  service.getServiceView("http://10.151.40.222:8080", "default", (data) => {
```

Comment:
do not hard code uri, use `webportalConfig` instead

Reply:
Solved in 4th commit.